libportal/camera.c

```diff
@@ -276,6 +284,27 @@ xdp_portal_access_camera_finish (XdpPortal     *portal,
   return g_task_propagate_boolean (G_TASK (result), error);
 }
 
+GVariant *
+xdp_portal_access_camera_finish_streams (XdpPortal     *portal,
+                                         GAsyncResult  *result,
+                                         GError       **error)
+{
+  g_return_val_if_fail (XDP_IS_PORTAL (portal), FALSE);
+  g_return_val_if_fail (g_task_is_valid (result, portal), FALSE);
+  g_return_val_if_fail (g_task_get_source_tag (G_TASK (result)) == xdp_portal_access_camera, FALSE);
+
+  if (g_task_propagate_boolean (G_TASK (result), error))
```

Comment:
Would it be necessary to retrieve `streams` before calling this? I think this could potentially drop the last reference to this GTask.